ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py

```diff
@@ -5,14 +5,8 @@
 from ssqatest.src.pages.Header import Header
 import time
 
-# from demostore_automation.src.pages.HomePage import HomePage
-# from demostore_automation.src.configs.MainConfigs import MainConfigs
-# from demostore_automation.src.pages.CartPage import CartPage
-# from demostore_automation.src.pages.Header import Header
-# import time
 import math
-# import pdb
-# pdb.set_trace()
+
 
 pytestmark=[pytest.mark.feregression, pytest.mark.fesmoke, pytest.mark.end_to_end]

```

Comment:
This is upto you but I dont think I would consider these tests 'end_to_end'. If it was me I would remove the 'end_to_end' marker.

Reply:
Yes you are correct. I removed the end_to_end mark. Will commit the test.